README.md

```diff
@@ -1,75 +1,5 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# FCC Test Suite Template
 
-# Project Name
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
```

Comment:
Delete the extra information in your "Readme" file. As this file shows the description of your project.

Reply:
- [ ]  Dear @Talha-Hanif5666, Your comments are great, But remember we need to be as polite as possible. This results in better communication and collaboration with colleagues. instead of starting with `Delete`, you could say `Please delete.